frontend-vue/src/components/primitives/button/util/LoadingWrapper.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div v-if="isLoading">
+    <div class="loader">
```

Comment:
I would prefer to use the utility classes from tailwind to keep the css consistent.

You can rewrite it with 
`inline-grid grid-flow-col w-full place-content-center justify-self-center`